### DISCUSSION TITLE
Remove css Imports from Esm Output

### DISCUSSION THREAD

**Discussion by vmani30** [parcel-bundler/parcel#9263] [Q&A]
I’m creating a transformer plugin where I'm adding additional dependencies using `asset.addURLdepenency` with `isLibrary` set to true, but I'm encountering an issue with the ESM output generation. 

The generated js file has an import statement for the css module, which breaks my static server:
`import "./styles.css" `

Is there any way to configure parcel or the css plugin to use a different method? Either stringify the contents or use link tags?

**Comment by ecjep24** [comment]
Same for me..
